_overviews/toolkit/munit-single-test.md

```diff
@@ -9,11 +9,103 @@ next-page: munit-clues
 
 {% include markdown.html path="_markdown/install-munit.md" %}
 
-<!-- 
+## Declare specific tests to run from within the test file
+
+Within a test suite file, you can select individual tests to run by appending `.only` to a test name, e.g.
+
+{% tabs 'only-demo' class=tabs-scala-version %}
+
+{% tab 'Scala 2' %}
+```scala
+class MathSuite extends munit.FunSuite {
+  test("addition") {
+    assert(1 + 1 == 2)
+  }
+
+  test("multiplication".only) {
+    assert(3 * 7 == 21)
+  }
+
+  test("remainder".only) {
+    assert(13 % 5 == 3)
+  }
+}
+```
+{% endtab %}
+{% tab 'Scala 3' %}
+```scala
+class MathSuite extends munit.FunSuite:
+  test("addition") {
+    assert(1 + 1 == 2)
+  }
+
+  test("multiplication".only) {
+    assert(3 * 7 == 21)
+  }
+
+  test("remainder".only) {
+    assert(13 % 5 == 3)
+  }
+```
+{% endtab %}
+
+{% endtabs %}
+
+In the above example, [running the tests as usual]({% link _overviews/toolkit/munit-test-suite.md %}) will only run the `"multiplication"` and `"remainder"` tests (i.e. `"addition"` is ignored). This is useful to quickly debug specific tests in a suite.
+
+### Alternative: exclude specific tests
```

Comment:
```suggestion
## Alternative: exclude specific tests
```

no?

Reply:
I intended it to be a sub section of the previous, but If you think the title is fine in isolation then ok